### PR TITLE
delete deprecated flag --force-pull

### PR DESCRIPTION
**Pull request by guangxuli**
/cc @bparees

### CODE REVIEW

**Comment by bparees** [comment]
it's been deprecrated since november 2015, i guess that's long enough.

File: pkg/cmd/util.go [hunk 1]
```diff
@@ -18,12 +18,12 @@ func AddCommonFlags(c *cobra.Command, cfg *api.Config) {
...
 	c.Flags().StringVar(&(cfg.CallbackURL), "callback-url", "",
 		"Specify a URL to invoke via HTTP POST upon build completion")
-	c.Flags().BoolVar(&(cfg.ForcePull), "force-pull", false,
-		"DEPRECATED: Always pull the builder image even if it is present locally")
 	c.Flags().VarP(&(cfg.BuilderPullPolicy), "pull-policy", "p",
 		"Specify when to pull the builder image (always, never or if-not-present)")
 	c.Flags().Var(&(cfg.PreviousImagePullPolicy), "incremental-pull-policy",
 		"Specify when to pull the previous image for incremental builds (always, never or if-not-present)")
+	c.Flags().Var(&(cfg.RuntimeImagePullPolicy), "runtime-pull-policy",
+		"Specify when to pull the runtime image (always, never or if-not-present)")
```
Comment (guangxuli) [inline pkg/cmd/util.go]: rebase addressed.

Comment (guangxuli) [inline pkg/cmd/util.go]: indeed :+1:

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
looks like some tests will need to be updated.

**Comment by guangxuli** [comment]
@bparees remove `--force-pull` from  **hack\test-stirunimage.sh**  `s2i rebuild rack-test-app:latest rack-test-app:v1 -p never --loglevel=5 &> "${WORK_DIR}/s2i-rebuild.log"`   
PTAL.

**Comment by bparees** [comment]
[merge]

**Pull request merged**